ffi/capi/src/collator.rs

```diff
@@ -31,7 +31,7 @@ pub mod ffi {
     // `ResolvedCollatorOptions` makes more sense as English.
     #[diplomat::rust_link(icu::collator::ResolvedCollatorOptions, Struct)]
     #[diplomat::out]
-    #[diplomat::attr(any(dart, js), rename = "ResolvedCollatorOptions")]
+    #[diplomat::attr(any(dart, js), rename = "CollatorResolvedOptions")]
```

Comment:
thought: (followup ok) ought this be `supports = non_exhaustive_structs`?

and below

fine with this happening in a different PR